cmd/kpod/login.go

```diff
@@ -0,0 +1,89 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/containers/image/pkg/authentication"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+	"golang.org/x/crypto/ssh/terminal"
+)
+
+const defaultServer = "https://registry-1.docker.io"
+
+var (
+	loginFlags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "password, p",
+			Usage: "Password for registry",
+		},
+		cli.StringFlag{
+			Name:  "username, u",
+			Usage: "Username for registry",
+		},
+	}
+	loginDescription = "Login to a container registry on a specified server." +
+		"Tha default registry is https://registry-1.docker.io"
```

Comment:
Typo: `Tha`

Reply:
Probably need some whitespace between these sentences.

Reply:
Fixed the typo. Why is the whitespace needed?

Reply:
How does 
./kpod login --help

Look?

Reply:
You can do multi line text variables by using `` for example kpod mount has:

```
	mountDescription = `
   kpod mount
   Lists all mounted containers mount points

   kpod mount CONTAINER-NAME-OR-ID
   Mounts the specified container and outputs the mountpoint
`
```